lighthouse-core/audits/accessibility/color-contrast.js

```diff
@@ -34,6 +34,9 @@ class ColorContrast extends AxeAudit {
       category: 'Accessibility',
       name: 'color-contrast',
       description: 'Background and foreground colors have a sufficient contrast ratio',
+      helpText: 'Some users with low vision have a difficult time sensing contrast. ' +
```

Comment:
Suggest rewording to "Low contrast text is difficult or impossible for many users to read."